components/base/ButtonLink.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <NuxtLink
+    class="
+      rounded
+      px-3
```

Comment:
We _can_, the thing that makes me hesitant about writing any of these paddings into the `class` is that it makes it hard to overwrite later on. Since I'm new to tailwind, I'm not sure if there's a good way to handle this or not. I might have to read more into tailwind though, cause this seems like it shouldn't be an issue.

Reply:
Okay, yeah. Reading more into the documentation, it looks like this definitely isn't an issue when done correctly. I'm for sure going to try and restructure these components even further to take advantage of the right way of doing things.
https://tailwindcss.com/docs/extracting-components

Reply:
That's a good point. I think that'll be a good way to go about things